docs/articles/intro/tutorial-1.md

```diff
@@ -0,0 +1,229 @@
+set the actor in a known-good state, which usually means a clean starting stage. **What actually happens though is
+that the `PreRestart()` and `PostRestart()` methods are called which, if not overridden, by default delegate to
+`PostStop()` and `PreStart()` respectively**. You can experiment with overriding these additional methods and see
+how the output changes.
+
+For the impatient, we also recommend looking into the [supervision reference page](xref:supervision) for more in-depth
+details.
+
+### The First Actor
+
+Actors are organized into a strict tree, where the lifecycle of every child is tied to the parent and where parents
+are responsible for deciding the fate of failed children. At first, it might not be evident how to map our problem
+to such a tree, but in practice, this is easier than it looks. All we need to do is to rewrite our architecture diagram
+that contained nested boxes into a tree:
+
+![actor tree diagram of the architecture](/images/arch_tree_diagram.png)
+
+In simple terms, every component manages the lifecycle of the subcomponents. No subcomponent can outlive the parent
+component. This is exactly how the actor hierarchy works. Furthermore, it is desirable that a component handles the failure
+of its subcomponents. Together, these two desirable properties lead to the conclusion that the "contained-in" relationship of components should be mapped to the
+"children-of" relationship of actors.
+
+The remaining question is how to map the top-level components to actors. It might be tempting to create the actors
+representing the main components as top-level actors. We instead, recommend creating an explicit component that
+represents the whole application. In other words, we will have a single top-level actor in our actor system and have
+the main components as children of this actor.
+
+The first actor happens to be rather simple now, as we have not implemented any of the components yet. What is new
+is that we have dropped using `Console.WriteLine()` and instead use `ILoggingAdapter` which allows us to use the
+logging facility built into Akka.NET directly. Furthermore, we are using a recommended creational pattern for actors; define a static `Props()` method in the the actor:
+
+[!code-csharp[Main](../../examples/Tutorials/Tutorial1/IotSupervisor.cs?name=iot-supervisor)]
+
+All we need now is to tie this up with a class with the `main` entry point:
+
+[!code-csharp[Main](../../examples/Tutorials/Tutorial1/IotApp.cs?name=iot-app)]
+
+This application does very little for now, but we have the first actor in place and we are ready to extend it further.
+
+## What is next?
+
+In the following chapters we will grow the application step-by-step:
+
+ 1. We will create the representation for a device
+ 2. We create the device management component
+ 3. We add query capabilities to device groups
+ 4. We add the dashboard component
```

Comment:
The dashboard component isn't described anywhere else. Is that still coming ? Or is it for later ?